docs: Grammar overhaul (#20636)
nuxt/nuxt: docs/1.getting-started/4.assets.md

# Assets

Nuxt uses two directories to handle assets like stylesheets, fonts, or images.

- The [`public/` directory](/docs/guide/directory-structure/public) content is served at the server root as-is.
- The [`assets/` directory](/docs/guide/directory-structure/assets) contains, by convention, every asset that you want the build tool (Vite or webpack) to process.

## `public/` Directory

> The [`public/` directory](/docs/guide/directory-structure/public) is a public server for static assets publicly available at a defined URL of your application.

Reviewer: ```suggestion
The [`public/` directory](/docs/guide/directory-structure/public) is a server for static assets publicly available at a defined URL of your application.
```

## `assets/` Directory

> Nuxt uses [Vite](https://vitejs.dev/guide/assets.html) or [webpack](https://webpack.js.org/guides/asset-management/) to build and bundle your application. The main function of these build tools is to process JavaScript files, but they can be extended through [plugins](https://vitejs.dev/plugins/) (for Vite) or [loaders](https://webpack.js.org/loaders/) (for webpack) to process other kinds of assets, like stylesheets, fonts or SVG. This step transforms the original file mainly for performance or caching purposes (such as stylesheet minification or browser cache invalidation).

Reviewer: ```suggestion
Nuxt uses [Vite](https://vitejs.dev/guide/assets.html) or [webpack](https://webpack.js.org/guides/asset-management) to build and bundle your application. The main function of these build tools is to process JavaScript files, but they can be extended through [plugins](https://vitejs.dev/plugins) (for Vite) or [loaders](https://webpack.js.org/loaders) (for webpack) to process other kinds of assets, like stylesheets, fonts or SVG. This step transforms the original file mainly for performance or caching purposes (such as stylesheet minification or browser cache invalidation).
```

> By convention, Nuxt uses the `assets/` directory to store these files, but there is no auto-scan functionality for it, and you can use any other name for it.

Reviewer: ```suggestion
By default, Nuxt uses the `assets/` directory to store these files. You can customize its name in the [Nuxt config file](/docs/api/configuration/nuxt-config#assets).
```

Author: Implemented

> In your application's code, you can reference a file located in the `assets/` directory using the `~/assets/` path.

Reviewer: @danielroe, is it true during build time? What about `~assets/`?
